README-AR.md

```diff
@@ -30,17 +30,17 @@
 
 هذه قائمة بالموارد للأشخاص الجدد في المساهمة في المصادر المفتوحة.
 
-إذا وجدت موارد إضافية ، يرجى المساهمة في طلب سحب.
+إذا وجدت موارد إضافية ، يرجى المساهمة بانشاء طلب السحب.
 
-إذا كانت لديك أسئلة أو تعليقات ، يرجى إنشاء مشكلة.
+إذا كانت لديك أسئلة أو تعليقات ، يرجى إنشاء عرض المشكلة.
 
 **محتويات**
 
 - [Contributing to Open Source in general](#المساهمة-في-المصادر-المفتوحة-بشكل-عام)
 - [Direct GitHub searches](#عمليات-البحث-المباشرة-على-github)
 - [Mozilla's contributor ecosystem](#النظام-البيئي-للمساهمين-في-mozilla)
 - [Useful articles for new Open Source contributors](#مقالات-مفيدة-للمساهمين-الجدد-في-المصادر-المفتوحة)
-- [Using Version Control](#باستخدام-التحكم-في-الإصدار)
+- [Using Version Control](#باستخدام نظام-التحكم-في-الإصدار)
```

Comment:
Thanks for this update. Please change this title from the actual section header below, near Line 130

Reply:
done with Line 130 and I added new contributions

Reply:
Can you also update this Markdown link to link correctly? Right now, the Markdown render is not correctly formatted.

```suggestion
- [Using Version Control](#باستخدام-نظام-التحكم-في-الإصدار)
```